packages/core/src/shared/logger/logger.ts

```diff
@@ -84,20 +84,21 @@ export function compareLogLevel(l1: LogLevel, l2: LogLevel): number {
 
 /**
  * Gets the logger if it has been initialized
- * @param type Gets the logger type:
- * * `'main'` or `undefined`: Main logger; default impl: logs to log file and log output channel
- * * `'channel'`: Channel Logger; default impl: logs to the `main` channels and the `AWS Toolkit` output channel
- * * `'debug'`: Debug Console Logger; default impl: logs to the `channel` channels and the currently-active VS Code Debug Console pane.
+ * the logger is of `'main'` or `undefined`: Main logger; default impl: logs to log file and log output channel
  */
-export function getLogger(type?: keyof typeof toolkitLoggers): Logger {
-    const logger = toolkitLoggers[type ?? 'main']
+export function getLogger(): Logger {
```

Comment:
Now that it doesn't take any parameters any more, can we make this a variable named `logger` instead of a `getLogger` function?

This would make the code so much cleaner like - `logger.info(....)` which seems like a natural extension of Winston or any other logger system.

Reply:
Indeed. In that case, we would need a function. Here I saw that the function doesn't take any inputs so I interpreted it as a constant.

Reply:
yes that is the reason, those changes will be implemented in the next pull request, thanks for asking!